mmv1/products/ces/Tool.yaml

```diff
@@ -103,6 +103,47 @@ properties:
         type: NestedObject
         description: Represents a select subset of an OpenAPI 3.0 schema object.
         properties:
+          - name: additionalProperties
+            type: String
+            description: |-
+              Optional. Defines the schema for additional properties allowed in an object.
```

Comment:
nit: there's no need to include it in the description whether its optional or required. the generation covers this for documentation